### DISCUSSION TITLE
Synthesis/degradation of targets or proteins

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1793] [Questions & Problems]
Hello, everyone.
When describing the synthesis/degradation of a target or protein, in the "Reactions" module of mobi, I have observed that many published papers are defined in a similar way to the following:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/760af3ff-355a-401e-b698-c3a9a7842226)
What does "volume" mean? What is the purpose?

**Comment by Yuri05** [comment]
It's a volume of the parent container (where the corresponding reaction takes place).
